raiden-ts/src/raiden.ts

```diff
@@ -71,6 +71,7 @@ import { RaidenError, ErrorCodes } from './utils/error';
 export class Raiden {
   private readonly store: Store<RaidenState, RaidenAction>;
   private readonly deps: RaidenEpicDeps;
+  public config: RaidenConfig;
```

Comment:
Can it also been modified by hand by `raiden.config = {}` and do we want to do that?

Reply:
It could, but it'd be replaced on next `config$` emit; I don't think we need to bother, SDK tries best practices, but if a developer wants to shoot their own foot, I'm not gonna stop them